Code/Framework/AzFramework/AzFramework/Spawnable/SpawnableEntitiesManager.cpp

```diff
@@ -280,7 +307,67 @@ namespace AzFramework
         constexpr bool allowDuplicateIds = false;
 
         return AZ::IdUtils::Remapper<AZ::EntityId, allowDuplicateIds>::CloneObjectAndGenerateNewIdsAndFixRefs(
-                &entityTemplate, templateToCloneMap, &serializeContext);
+            &entityTemplate, templateToCloneMap, &serializeContext);
+    }
+
+    AZ::Entity* SpawnableEntitiesManager::CloneSingleAliasedEntity(
+        const AZ::Entity& entityTemplate,
+        const Spawnable::EntityAlias& alias,
+        EntityIdMap& templateToCloneMap,
+        AZ::Entity* previouslySpawnedEntity,
+        AZ::SerializeContext& serializeContext)
+    {
+        using ResultType = AZStd::pair<AZ::Entity*, uint32_t>;
+
+        AZ::Entity* clone = nullptr;
+        switch (alias.m_aliasType)
+        {
+        case Spawnable::EntityAliasType::Original:
+            // Behave as the original version.
+            clone = CloneSingleEntity(entityTemplate, templateToCloneMap, serializeContext);
```

Comment:
If the alias type is original than `alias.m_spawnable` is null. An asset cannot point to itself because it would cause a memory leak. When an asset refers to itself it will keep a reference count to itself at all times so the asset never gets cleared.